packages/node_modules/@node-red/nodes/core/parsers/70-CSV.js

```diff
@@ -19,9 +19,9 @@ module.exports = function(RED) {
     function CSVNode(n) {
         RED.nodes.createNode(this,n);
         this.template = (n.temp || "");
-        this.sep = (n.sep || ',').replace("\\t","\t").replace("\\n","\n").replace("\\r","\r");
+        this.sep = (n.sep || ',').replaceAll("\\t","\t").replaceAll("\\n","\n").replaceAll("\\r","\r");
```

Comment:
```suggestion
        this.sep = (n.sep || ',').replace(/\\t/g,"\t").replace(/\\n/g,"\n").replace(/\\r/g,"\r");
```
Will need testing and the other line changed to match as I'm tapping this out on my phone over a big slice of cake.